src/parsec_client/operations/service_api_coverage.md

```diff
@@ -86,77 +91,77 @@ cryptographic operations they could be used in.
-| Raw PKCS#1 v1.5 signature                  | ❌                  | ❌              | ❌              |
-| RSA PSS signature with hashing             | ❌                  | ✅              | ❌              |
-| ECDSA signature with hashing               | ❌                  | ❌              | ✅              |
-| ECDSA signature without hashing            | ❌                  | ❌              | ❌              |
-| Deterministic ECDSA signature with hashing | ❌                  | ❌              | ❌              |
+| Algorithm \ Provider                       | Mbed Crypto | PKCS 11 | TPM 2.0 | Trusted Service | CryptoAuth library |
+|--------------------------------------------|-------------|---------|---------|-----------------|--------------------|
+| RSA PKCS#1 v1.5 signature with hashing     | ✅         | ✅     | ✅     | ✅             | ❌                |
+| Raw PKCS#1 v1.5 signature                  | ❌         | ❌     | ❌     | ❌             | ❌                |
+| RSA PSS signature with hashing             | ❌         | ✅     | ❌     | ❌             | ❌                |
+| ECDSA signature with hashing               | ❌         | ❌     | ✅     | ❌             | ❌                |
+| ECDSA signature without hashing            | ❌         | ❌     | ❌     | ❌             | ❌                |
+| Deterministic ECDSA signature with hashing | ❌         | ❌     | ❌     | ❌             | ❌                |
 
 #### Asymmetric encryption algorithms
 
-| Algorithm                  | Mbed Crypto provider | PKCS 11 provider | TPM 2.0 provider |
-|----------------------------|----------------------|------------------|------------------|
-| RSA PKCS#1 v1.5 encryption | ✅                  | ✅              | ✅              |
-| RSA OAEP encryption        | ✅                  | ✅              | ✅              |
+| Algorithm \ Provider       | Mbed Crypto | PKCS 11 | TPM 2.0 | Trusted Service | CryptoAuth library |
+|----------------------------|-------------|---------|---------|-----------------|--------------------|
+| RSA PKCS#1 v1.5 encryption | ✅         | ✅     | ✅     | ✅             | ❌                |
+| RSA OAEP encryption        | ✅         | ✅     | ✅     | ✅             | ❌                |
 
 #### Key agreement algorithms
 
-| Algorithm | Mbed Crypto provider | PKCS 11 provider | TPM 2.0 provider |
-|-----------|----------------------|------------------|------------------|
-| FFDH      | ✅                  | ❌              | ❌              |
-| ECDH      | ✅                  | ❌              | ❌              |
+| Algorithm \ Provider | Mbed Crypto | PKCS 11 | TPM 2.0 | Trusted Service | CryptoAuth library |
+|----------------------|-------------|---------|---------|-----------------|--------------------|
+| FFDH                 | ✅         | ❌     | ❌     | ✅             | ❌                |
+| ECDH                 | ✅         | ❌     | ❌     | ✅             | ❌                |
```

Comment:
I would hold-off from making `Trusted Service` provider fully on-par with `Mbed Crypto` provider for the simple reason that since the operations aren't implemented, you can't really say we support `FFDH` or `ECDH`... I think the ones for which we have support on both sides (just asymmetric signing and verification, I guess...) should be the only ones fully equal 😓